docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3'
+
+services:
+  planner:
+    build: .
+    image: fabric8-planner-runtime
+    # command: npm run test:unit
+    environment:
+      API_URL: http://core:8080/api/
+    depends_on:
+      - core
+    networks:
+      - default
+    ports:
+      - "8088:8080"
+    volumes:
+      - ./runtime/dist:/dist:Z
+  core:
+    image: fabric8/fabric8-wit:latest
+    command: -config /usr/local/wit/etc/config.yaml
```

Comment:
Where is this file?

Reply:
`https://github.com/fabric8-services/fabric8-wit/blob/master/config.yaml`

Reply:
So is this a path inside the container?